tools/buildsteps/windows/download-dependencies.bat

```diff
@@ -48,7 +48,7 @@ IF NOT EXIST %FORMED_OK_FLAG% (
   ECHO.
   ECHO I tried to get the packages from %KODI_MIRROR%;
   ECHO if this download mirror seems to be having problems, try choosing another from
-  ECHO the list on http://mirrors.kodi.tv/list.html, and setting %%KODI_MIRROR%% to
+  ECHO the list on http://mirrors.kodi.tv/timestamp.txt?mirrorlist, and setting %%KODI_MIRROR%% to
```

Comment:
If there is a hard limit on 80 characters then I'll reformat this. Just let me know.